scripts/superdesk-search/react.js

```diff
@@ -1073,7 +1073,11 @@
 
                         takepackage: function(props) {
                             var item = props.item;
-                            var isTake = _.contains(['text'], item.type) && item.takes && item.takes.sequence > 1;
+                            var archiveItem = props.item.archive_item;
```

Comment:
Very minor, but for consistency I would do:
`var archiveItem = item.archive_item;`